src/fswAlgorithms/attGuidance/prescribedRot2DOF/_UnitTest/test_prescribedRot2DOF.py

```diff
@@ -0,0 +1,323 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+
+#
+#   Unit Test Script
+#   Module Name:        prescribedRot2DOF
+#   Author:             Leah Kiner
+#   Creation Date:      Nov 27, 2022
+#
+
+import pytest
+import inspect
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+from Basilisk.architecture import bskLogging
+from Basilisk.architecture import messaging
+from Basilisk.fswAlgorithms import prescribedRot2DOF  # import the module that is to be tested
+from Basilisk.utilities import RigidBodyKinematics as rbk
+from Basilisk.utilities import SimulationBaseClass
+from Basilisk.utilities import macros
+from Basilisk.utilities import unitTestSupport
+
+filename = inspect.getframeinfo(inspect.currentframe()).filename
+path = os.path.dirname(os.path.abspath(filename))
+bskName = 'Basilisk'
+splitPath = path.split(bskName)
+
+# Parametrize the user-configurable variables
+@pytest.mark.parametrize("thetaInit", [0.0])
```

Comment:
Please add a non-zero initial angle to make sure this works as well, maybe 0.01?

Reply:
Done.